.gitignore

```diff
@@ -0,0 +1 @@
+frontend/cfd-partner-finder-frontend/package-lock.json
```

Comment:
I think we want the lock file in git history.

https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore